cmd/antrea-controller/controller.go

```diff
@@ -168,8 +171,12 @@ func run(o *Options) error {
 		groupStore)
 
 	var externalNodeController *externalnode.ExternalNodeController
+	var bundleCollectionController *supportbundlecollection.Controller
```

Comment:
Maybe we should add another feature gate for support bundle collection?

Reply:
Sure, I will add it.